package-lock.json

```diff
@@ -461,8 +461,8 @@
             "integrity": "sha512-bzHZN8Pn+gS7DQA6n+iUmBfl0hO5DJq++QP3U6uTucDtk/0iGpXd/Gg7CGR0p8tJhofJyaKoWBuJI4eAO00BBg=="
         },
         "las2peer-frontend-user-widget": {
-            "version": "github:rwth-acis/las2peer-frontend-user-widget#63ad251f95b323938b78e0d68b14ec67a1a05511",
-            "from": "github:rwth-acis/las2peer-frontend-user-widget#0.3.0",
+            "version": "github:rwth-acis/las2peer-frontend-user-widget#8d98f79917eac4ba6c5ded0ec810308041ed8cd2",
+            "from": "github:rwth-acis/las2peer-frontend-user-widget#widget-position-fix",
```

Comment:
Package-lock.json seems outdated, as you specified another repository version in package.json